var/spack/repos/builtin/packages/nano/package.py

```diff
@@ -15,6 +15,7 @@ class Nano(AutotoolsPackage):
     list_depth = 1
 
     # 6.x
+    version("7.2", sha256="86f3442768bd2873cec693f83cdf80b4b444ad3cc14760b74361474fc87a4526")
```

Comment:
Ok, nitpick, but given the comment this one should be under `#7.x` (or we can delete the comments)

Reply:
Fixed!